### ISSUE TITLE
[1.17.1] OpenGL 1282 error while Overwolf overlay is enabled

### ISSUE THREAD

**Issue by NightscaleReaper** [sp614x/optifine#6313]
## Description of Issue
While Overwolf overlay is enabled for minecraft, OpenGL error 1282 appears in chat every ~6 seconds.

## Steps to Reproduce
1. Install Overwolf and have overlay enabled for minecraft under "Overlay and Hotkeys" in the Overwolf app's settings.

## OptiFine Version
1.17.1-OptiFine_HD_U_G9

## Installation Method

Installed with standalone installer (Optifine Installer).

## Log Files/Crash Reports
[latest.log](https://github.com/sp614x/optifine/files/7357710/latest.log)

## F3 Debug Screenshot
![2021-10-16_15 28 32](https://user-images.githubusercontent.com/92635703/137591414-16574bbe-7dc6-493c-8a77-b0d17c578630.png)

## Prior Testing
DOES only happen in Optifine version for 1.17.1. Vanilla works as intended.

## Additional Information
Was able to recreate this issue on another computer. Same symptoms, same problem.

**Comment by Medatur76** [comment]
[My Error Thread](https://github.com/sp614x/optifine/issues/6317)

**Comment by NightscaleReaper** [comment]
> Does it happen without Outplayed? I have the error with Outplayed too

Yeah since you need the overlay for it to be displayed or be used at all.
> [My Error Thread](https://github.com/sp614x/optifine/issues/6317) On thing to consider is updating your diver. I dont have the link but somewhere in the NVIDIA page is where you download new drivers

I HIGHLY doubt it's a GPU driver issue, it's more of a conflict between OpenGL and the overlay.

**Comment by NightscaleReaper** [comment]
> No, I just tested it. I set it to manual so it wont record me but still get the error. Its something else

Oh ok, I don't use Outplayed so I was just making an assumption. I have no idea what is causing your problem then.

**Comment by Medatur76** [comment]
O, I saw outplayed. I didn't see you said overwolf
sry.
Good luck on a solution
